Jenkinsfile

```diff
@@ -466,67 +504,99 @@ pipeline {
             expression { return env.BUILD_DOCS != "false" }
           }
           steps {
-            withBeatsEnv(){
-              makeTarget("Docs", "docs")
-            }
+            makeTarget("Docs", "docs")
           }
         }
       }
     }
   }
 }
 
-def makeTarget(context, target, clean = true){
+def makeTarget(context, target){
+  withGithubNotify(context: "${context}") {
+    withBeatsEnv(){
+      sh(label: "Make ${target}", script: """
+        eval "\$(gvm use ${GO_VERSION} --format=bash)"
+        make ${target}
+      """)
+    }
+  }
+}
+
+def mageTargetWin(context, target){
   withGithubNotify(context: "${context}") {
-    sh(label: "Make ${target}", script: "make ${target}")
+    withBeatsEnvWin(){
+      bat(label: "Mage ${target}", script: """
+        set
+        mage ${target}
+      """)
+    }
   }
 }
 
 def withBeatsEnv(Closure body){
   withEnv([
     "HOME=${env.WORKSPACE}",
     "GOPATH=${env.WORKSPACE}",
-    "PATH=${env.WORKSPACE}/bin:${env.PATH}",
+    "PATH+GO=${env.WORKSPACE}/bin:${env.PATH}",
```

Comment:
What does `PATH+GO` do? Is that something special?

Reply:
it is a way to set PATH=$PATH+":/new_path"